charts/portworx/templates/hooks/post-install/px-create-cluster-token.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if (.Values.api.enabled) and (.Values.dynamicProvision.enabled) }}
+{{- $customRegistryURL := .Values.customRegistryURL | default "none" }}
+  {{- $registrySecret := .Values.registrySecret | default "none" }}
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: kube-system
+  name: px-set-cluster-token
```

Comment:
We don't want to run this job by default as using cluster-wide tokens is just one mechanism to provide secrets to PX. Many users only prefer to use per-volume encryption. Looks like this change is setting cluster-token for all users.